src/actions/game.js

```diff
@@ -43,6 +43,16 @@ const assignCardsToPlayers = (deck, players) => {
   );
 };
 
+const handleGameUpdates = g => (dispatch) => {
+  const { gameOver } = g;
+
+  dispatch(setGame(g));
+
+  if (gameOver) {
```

Comment:
I'm not sure if I'm missing something, but I don't see you clearing the game, this could lead to unintended behavior.

Also, we're already handling the navigation to `Lobby` when `party.gameInProgress` changes to `false` [here](https://github.com/lycan-city/werewolf-virtual-cards/blob/master/src/actions/party.js#L32)